web/src/components/auth/login/AuthLoginEmailPasswordForm.tsx

```diff
@@ -20,11 +20,55 @@ export function EmailPasswordForm({
 }) {
   const router = useRouter();
   const { popup, setPopup } = usePopup();
-  const [isWorking, setIsWorking] = useState(false);
+  const [isPending, setIsPending] = useState(false);
+
+  async function requestVerificationForSignup (values: { email: string; password: string; }) {
+    setIsPending(true);
+    const response = await basicSignup(values.email, values.password);
+    if (!response.ok) {
+      const errorDetail = (await response.json()).detail;
+
+      let errorMsg = "Unknown error";
```

Comment:
Only 2 errors possible?
@ptitzlabs do you know if there can be more?

Reply:
I propose to highlight separate issue. Because we need to do it not only here.